src/content/docs/enterprise/tasks/tasks.mdx

```diff
@@ -206,6 +207,37 @@ Sequential tasks are available for the following task types:
 * Pending proofreading tasks can be deleted if necessary. In this case, linked translation tasks are preserved.
 * Deleting a translation task with an associated sequential proofreading task results in the deletion of both tasks simultaneously.
 
+## Task Cross-Review :badge[New]{variant=success} {#task-cross-review}
+
+The **Task Cross-Review** feature helps ensure proper oversight in task workflows by requiring approval before contributors can start working. It’s especially beneficial for teams with internal approval processes, such as budget approvals, where tasks must be reviewed and approved by designated managers or the owner. By adding an extra layer of accountability, this feature ensures tasks don’t move forward without proper review.
+
+<Image src={pendingCrossReviewTasks} alt="Pending Cross-Review Task" />
+
+### How Task Cross-Review Works
+
+When **Task Cross-Review** is enabled:
+
+* During task creation, a notification informs the user that the task will require approval from eligible project members (selected managers or the owner) before assigned members can start working.
+
+* On the Board, tasks awaiting approval display an alert indicating they are pending review from selected managers or the owner, making them easy to identify.
+
+* In the task details view, an alert states, *"The task is currently pending review from selected managers or the owner,"* and the task is labeled with the status **For Approval**.
+
+Only selected managers or the owner can approve the task. The manager who created the task cannot approve it, but the owner can always approve their own tasks. Once approved, the task status changes to **To Do**, indicating that contributors can begin work.
+
+### Configuring Task Cross-Review
+
+To enable and configure Task Cross-Review, follow these steps:
+
+<Steps>
+  1. Open your project and go to **Settings > Privacy**.
+  2. Enable the **Task Cross-Review** option in the **Tasks** section.
+  3. Select at least one manager responsible for approving tasks. You can search for managers by name, username, or email.
+  5. Click **Save** to apply the changes.
+</Steps>
+
+New tasks created after enabling **Task Cross-Review** will require approval, while existing tasks remain unaffected.
```

Comment:
Better to move it to the Privacy Settings page and link the two sections together